### PR TITLE
Windows changes for 0.4.1

### PR DESCRIPTION
**Pull request by peterjc123**

### CODE REVIEW

File: packaging/windows/internal/cuda_install.bat [hunk 1]
```diff
@@ -53,6 +54,23 @@ if not exist "%SRC_DIR%\temp_build\cudnn-10.0-windows10-x64-v7.4.1.5.zip" (
 
 goto cuda_common
 
+:cuda101
+
+if not exist "%SRC_DIR%\temp_build\cuda_10.1.243_426.00_win10.exe" (
+    curl -k -L http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_426.00_win10.exe --output "%SRC_DIR%\temp_build\cuda_10.1.243_426.00_win10.exe"
```
Comment (peterjc123) [inline packaging/windows/internal/cuda_install.bat]: +1

File: packaging/windows/internal/cuda_install.bat [hunk 2]
```diff
@@ -53,6 +54,23 @@ if not exist "%SRC_DIR%\temp_build\cudnn-10.0-windows10-x64-v7.4.1.5.zip" (
...
+    if errorlevel 1 exit /b 1
+    set "CUDA_SETUP_FILE=%SRC_DIR%\temp_build\cuda_10.1.243_426.00_win10.exe"
+    set "ARGS=nvcc_10.1 cuobjdump_10.1 nvprune_10.1 cupti_10.1 cublas_10.1 cublas_dev_10.1 cudart_10.1 cufft_10.1 cufft_dev_10.1 curand_10.1 curand_dev_10.1 cusolver_10.1 cusolver_dev_10.1 cusparse_10.1 cusparse_dev_10.1 nvgraph_10.1 nvgraph_dev_10.1 npp_10.1 npp_dev_10.1 nvrtc_10.1 nvrtc_dev_10.1 nvml_dev_10.1"
+)
+
+if not exist "%SRC_DIR%\temp_build\cudnn-10.1-windows10-x64-v7.6.4.38.zip" (
+    curl -k -L https://www.dropbox.com/s/i3aur3wh9lykmgx/cudnn-10.1-windows10-x64-v7.6.4.38.zip?dl=1 --output "%SRC_DIR%\temp_build\cudnn-10.1-windows10-x64-v7.6.4.38.zip"
```
Comment (peterjc123) [inline packaging/windows/internal/cuda_install.bat]: +1

Comment (yf225) [reply packaging/windows/internal/cuda_install.bat]: I uploaded it to https://ossci-windows.s3.amazonaws.com/cudnn-10.1-windows10-x64-v7.6.4.38.zip

File: packaging/windows/internal/cuda_install.bat (hunk 1 above)
Comment (yf225) [reply packaging/windows/internal/cuda_install.bat]: I uploaded it to https://ossci-windows.s3.amazonaws.com/cuda_10.1.243_426.00_win10.exe

**Pull request merged**